code/style.css

```diff
@@ -1,7 +1,295 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Murecho', sans-serif;
+ -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
+}
+
+body {
+  background-color: black;
+  color: white;
+}
+
+/*Hero*/
+.video-hero {
+  height: 40vh;
+  width: 100vw;
+  position: relative;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  overflow: hidden;
+}
+
+.video-hero video {
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  object-fit: cover;
+  left: 0;
+  top: 0;
+}
+
+.text-hero {
+  position: absolute;
+  color: white;
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  padding-left: 10%;
+  padding-bottom: 100px;
+}
+
+h1 {
+  font-size: 40px;
+}
+
+.button-sub {
+  display: flex;
+  flex-direction: column;
+  align-self: flex-start;
+  padding-top: 150px;
+  padding-left: 25%;
+  margin: 20px;
+}
+
+.sub-button-style {
```

Comment:
Made me wonder what's more accurate; to have the entire css document in chronological order, matching the html document, or collect it by type like you did here? I don't know.